### ISSUE TITLE
[Order Creation] Accessibility → Add Customer → Details, Billing Address, etc. are not marked as headings

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-android#5815]
The section titles like Details and Billing Address are not dictated by TalkBack as headings. This means that it will not be possible to navigate using the Headings reader control. 

<img src="https://user-images.githubusercontent.com/198826/152883075-f96ede77-09d8-4acc-b632-eb115b690368.png" width="300">

**Comment by rossanafmenezes** [comment]
👋  Hi @shiki 

It looks like the headings in the Orders screen are not read as headings either. Are we setting these as headings as well?

<img src="https://user-images.githubusercontent.com/30724184/157673535-6f915dde-c546-407e-85cf-4ef2e110f5f5.jpeg" width="350"/>

**Comment by shiki** [comment]
Good catch, @rossanafmenezes. 🙂 Yep, I believe those should be headings too.

**Comment by rossanafmenezes** [comment]
Awesome. Will work on that too. Thanks for confirming it. 😃
